docs/sdk/src/reference_docs/chain_spec_runtime/tests/chain_spec_builder_tests.rs

```diff
@@ -0,0 +1,128 @@
+use serde_json::{json, Value};
+use std::{process::Command, str};
+
+const WASM_FILE_PATH: &str =
+	"../../../../../target/release/wbuild/chain-spec-guide-runtime/chain_spec_guide_runtime.wasm";
+
+const CHAIN_SPEC_BUILDER_PATH: &str = "../../../../../target/release/chain-spec-builder";
+
+fn get_chain_spec_builder_path() -> &'static str {
+	// dev-dependencies do not build binary. So let's do the naive work-around here:
+	let _ = std::process::Command::new("cargo")
+		.arg("build")
+		.arg("--release")
+		.arg("-p")
+		.arg("staging-chain-spec-builder")
+		.arg("--bin")
+		.arg("chain-spec-builder")
+		.status()
+		.expect("Failed to execute command");
+	CHAIN_SPEC_BUILDER_PATH
+}
+
+#[test]
+#[docify::export]
+fn list_presets() {
+	let output = Command::new(get_chain_spec_builder_path())
+		.arg("list-presets")
+		.arg("-r")
+		.arg(WASM_FILE_PATH)
+		.output()
+		.expect("Failed to execute command");
+
+	let output: serde_json::Value = serde_json::from_slice(&output.stdout).unwrap();
+
+	let expected_output = json!({
+		"presets":[
+			"preset_1",
+			"preset_2",
+			"preset_3",
+			"preset_4"
+		]
+	});
+	assert_eq!(output, expected_output, "Output did not match expected");
+}
+
+#[test]
+#[docify::export]
+fn get_preset() {
+	let output = Command::new(get_chain_spec_builder_path())
+		.arg("display-preset")
+		.arg("-r")
+		.arg(WASM_FILE_PATH)
+		.arg("-p")
+		.arg("preset_1")
+		.output()
+		.expect("Failed to execute command");
+
+	let output: serde_json::Value = serde_json::from_slice(&output.stdout).unwrap();
+
+	//note: copy of chain_spec_guide_runtime::preset_1
+	let expected_output = json!({
+		"bar": {
+			"initialAccount": "5CiPPseXPECbkjWCa6MnjNokrgYjMqmKndv2rSnekmSK2DjL",
+		},
+		"foo": {
+			"someEnum": {
+				"Data2": {
+					"v": "0x0c0f"
+				}
+			},
+			"someInteger": 100
+		},
+	});
+	assert_eq!(output, expected_output, "Output did not match expected");
+}
+
+#[test]
+#[docify::export]
```

Comment:
In general, I think some of the docify export snippets are too long. For example, for this one, should not just export the `Command`? the rest of it is just for our testing.

Reply:
My intention here was also to show the output of the command. I think it has some educative value.

Ideally if we could have macro like this:
```
command_line_sample!({
  ls -1
}, {
  output_file1,
  output file2 
})
```